api-spec/Fields-fragment.yaml

```diff
@@ -0,0 +1,63 @@
+paths:
+  /stac/search:
+    get:
+      parameters:
+        - $ref: '#/components/parameters/fields'
+components:
+  parameters:
+    fields:
+      name: fields
+      in: query
+      description: |
+        The optional fields parameter defines the shape of the output features. 
+
+        The geometry member determines whether the geometry is populated or is null. The
+        include and exclude members specify an array of property names that are either 
+        included or excluded from the result, respectively. If both include and exclude 
+        are specified, include takes precedence.
+      required: false
+      schema:
+        type: object
+      properties:
```

Comment:
This should be indented one level. Also I think you can replace the schema with a ref potentially.